src/js/edu-benefits/5495/helpers.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import _ from 'lodash/fp';
+import { transformForSubmit } from '../../common/schemaform/helpers';
+
+export const bankAccountChangeLabels = {
+  startUpdate: 'Start or update direct deposit',
+  stop: 'Stop direct deposit',
+  noChange: 'No change to payment method'
+};
+
+export function transform(formConfig, form) {
+  // All the king's horses and all the king's men
```

Comment:
Do we know we'll need this stuff for these forms? I know one of the forms needed it, but not all. if not sure, would just stringify the json and return for now.